BuildResidentialHPXML/measure.rb

```diff
@@ -3253,6 +3253,11 @@ def validate_hpxml(runner, hpxml_path, hpxml_doc)
 
 class HPXMLFile
   def self.create(runner, model, args, epw_file)
+    if (args[:hvac_control_heating_season_period].is_initialized && args[:hvac_control_heating_season_period].get == HPXML::BuildingAmerica) || (args[:hvac_control_cooling_season_period].is_initialized && args[:hvac_control_cooling_season_period].get == HPXML::BuildingAmerica) || (args[:apply_defaults].is_initialized && args[:apply_defaults].get)
```

Comment:
This is certainly correct, though it could be simplified from
`foo.is_initialized && foo.get == XXXX`
to
`foo.to_s == XXXX`